src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -638,7 +642,13 @@ end
 --- @see fg
 --- @see showColors
 function bg(console, colorName)
-  local colorName = colorName or console
+  assert(type(console) == 'string', 'bg: bad argument #1 type (expected string, got '..type(console)..'!)')
```

Comment:
You ought to establish whether you have one or two arguments so you can report whether this should have been a window/console name or a colour name, IMHO.

Reply:
Not part of original scope: was broken, is still broken